src/App.js

```diff
@@ -300,54 +355,76 @@ class App extends React.Component {
     if (fatalError !== null) {
       throw fatalError
     }
+    const { address: identityAddress = null, label: identityLabel = '' } =
```

Comment:
```suggestion
    const { address: intentAddress = null, label: intentLabel = '' } =
```

I think signifying that it's a user intention is useful. And we're still destructuring `identityIntent` so we know it's an `identity`